### DISCUSSION TITLE
Breaking change in patch version?

### DISCUSSION THREAD

**Discussion by markoposavec** [cloudprober/cloudprober#718] [Q&A] [answered]
This change breaks how reloading of certificates works for grpc server. I assume the same update method cannot be used in both contexts.
https://github.com/cloudprober/cloudprober/pull/697

**Comment by manugarg** [comment, marked as answer]
Thanks for flagging this @markoposavec. https://github.com/cloudprober/cloudprober/pull/719 should fix the issue.
